runtime/src/iree/base/alignment.h

```diff
@@ -115,6 +115,17 @@ static inline bool iree_device_size_has_alignment(
   return iree_device_align(value, alignment) == value;
 }
 
+// Returns true if |value| is a power-of-two.
+static inline bool iree_is_power_of_two_uint64(uint64_t value) {
+  return (value != 0) && ((value & (value - 1)) == 0);
+}
```

Comment:
Doesn't look like we have tests for this file. Math code would be pretty easy to add coverage for :p

(This looks correct, but a sanity check would be nice)

Oh also, adding tests for functions like this could help mark code as live for cleanup changes like https://github.com/openxla/iree/pull/15471 (`-Wno-unused-function`).

Reply:
yeah, would be good to backfill the tests in here. this is just iree_host_size_is_power_of_two / iree_device_size_is_power_of_two with uint64_t instead of a dynamically sized value, so it has had a lot of coverage.